src/common/enums/errorMessages.ts

```diff
@@ -7,4 +7,10 @@ export const errorMessages = {
   VALIDATION_ERROR: 'The validation has failed.',
   ENTITY_NOT_FOUND: (entity: string, id: string) =>
     `${entity} with id ${id} was not found.`,
+  BAD_REQUEST_LOGIN_ERROR: 'Please provide email or password.',
```

Comment:
Seems more logical this way, given you code:
```suggestion
  BAD_REQUEST_LOGIN_ERROR: 'Please provide email and password.',
```

Reply:
done